### ISSUE TITLE
Forum error when not logged in.

### ISSUE THREAD

**Issue by JoshRickers** [Schamper/nodebb-plugin-shoutbox#72]
I have noticed that if i comment out the following code in the config.js that the error stops. I am guessing that this is because there is no user data for a user who is not logged in.

```
	if (!data || !data.uid) {
		return callback(new Error('Invalid data'));
	}
```

A hacky fix.

**Comment by Ribesg** [comment]
Please fix this ASAP.

**Comment by Schamper** [comment]
It's open source, you're free to contribute. I work on this whenever I can, it just so happens that I don't have a lot of free time lately.

**Comment by JoshRickers** [comment]
Or @Ribesg could implement the hacky fix above. We have had no issues with it commented out over at http://BitBangers.co.uk

**Comment by Ribesg** [comment]
> It's open source, you're free to contribute. I work on this whenever I can, it just so happens that I don't have a lot of free time lately.

Sorry for the misunderstanding, I meant that this should be the first issue to work on once you get time for it. I of course applied the (ugly :p) fix provided by @JoshRickers for now.

I understand that you don't have time, see, I don't have time to deal with it myself either :-/

**Comment by Firelad97** [comment]
When I do this, shoutbox isn't showing up.

**Comment by Schamper** [comment]
Fixed with v0.3.0.
